src/shared/utils/index.js

```diff
@@ -223,12 +223,12 @@ export const getAxisLabelsSeries = ({
   yScale: yScaleSpec,
   width,
   height,
-  axisBottomLabelValues,
+  axisBottomTickValues,
```

Comment:
line and scatter are the only components calling this function and they use `axisBottomTickValues` not `axisBottomLabelsValues`

https://github.com/EQWorks/chart-system/blob/423f158afbb81e1d6a0d9c3d22d5dc893da59447/src/components/scatter-chart/index.js#L67-L82

https://github.com/EQWorks/chart-system/blob/423f158afbb81e1d6a0d9c3d22d5dc893da59447/src/components/line-chart/index.js#L82-L97